scripts/tests/run_tv_casting_test.py

```diff
@@ -96,13 +103,19 @@ def extract_value_from_string(line: str) -> str:
 
     The string is expected to be in the following format as it is received
     from the Linux tv-casting-app output:
+    \x1b[0;34m[1715206773402] [20056:2842184] [DMG]  Cluster = 0x506,\x1b[0m
+    The substring to be extracted here is '0x506'.
+    Or:
     \x1b[0;34m[1713741926895] [7276:9521344] [DIS] Vendor ID: 65521\x1b[0m
     The integer value to be extracted here is 65521.
     Or:
     \x1b[0;34m[1714583616179] [7029:2386956] [SVR] 	device Name: Test TV casting app\x1b[0m
     The substring to be extracted here is 'Test TV casting app'.
     """
-    value = line.split(':')[-1].strip().replace('\x1b[0m', '')
+    if '=' in line:
+        value = line.split('=')[-1].strip().replace(',\x1b[0m', '')
+    else:
+        value = line.split(':')[-1].strip().replace('\x1b[0m', '')
```

Comment:
It probably makes sense to check if we found ":" in line before trying to split by it and then add an else case where we did not find either separator, which should be considered an error case.

Reply:
Addressed in latest commit.